templates/store-lists-loop.php

```diff
@@ -13,13 +13,14 @@
                     $store_phone              = $vendor->get_phone();
                     $store_info               = dokan_get_store_info( $seller->ID );
                     $store_address            = dokan_get_seller_short_address( $seller->ID );
-                    $store_banner_url         = $store_banner_id ? wp_get_attachment_image_src( $store_banner_id, $image_size ) : DOKAN_PLUGIN_ASSEST . '/images/default-store-banner.png';
+                    $default_store_banner     = DOKAN_PLUGIN_ASSEST . '/images/default-store-banner.png';
+                    $store_banner_url         = $store_banner_id ? wp_get_attachment_image_src( $store_banner_id, $image_size ) : dokan_get_option( 'default_store_banner', 'dokan_appearance', $default_store_banner );
```

Comment:
Kindly contact me, I have a better solution for this. we can rewrite existing Vendor class methods to achieve the same effect.